frontend/src/pages/Registration/Registration.tsx

```diff
@@ -26,24 +30,34 @@ export default class Registration extends React.Component {
                     <label>Password</label>
                     <input type="text" />
 
-                    <select id="status" name="status">
-                        <option value="online">Online</option>
-                        <option value="offline">Offline</option>
-                        <option value="dnd">Do not disturb</option>
-                    </select>
-
-                    <input type="checkbox" id="remember" name="remember" value="remember" />
-                    <label htmlFor="remember">Remember me</label>
-
-                    <input type="checkbox" id="password" name="password" value="password" />
-                    <label htmlFor="password">Remember my password</label>
-
-                    <input type="checkbox" id="automatic" name="automatic" value="automatic" />
-                    <label htmlFor="automatic">Sign me in automatically</label>
-
-                    <button type="submit" value="Submit">Submit</button>
+                    <span>
+                        <label>Status</label>
+                        <select id="status" name="status">
+                            <option value="online">Online</option>
+                            <option value="offline">Offline</option>
+                            <option value="dnd">Do not disturb</option>
+                        </select>
+                    </span>
+                    
+                    <span>
```

Comment:
Technically this should be a div :P but its fine. Lets keep the hackathon spirit